content/participants/ttjc@Mohan6677/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Mohan Tummala
+type: "participant"
+name: "Tummala Mohan Sai"
+portfolio: ""
+github-link: "https://github.com/Mohan6677"
+linkedin-link: "https://www.linkedin.com/in/mohan-tummala-76191a171/"
+looking-for: "internship"
+description: "sophomore at NIT Kurukshetra"
+---
+
+Hi! I am a student let me walk you through my intro, projects and blogs.
```

Comment:
Hey! @Mohan6677 instead of writing I am a **student** you can use **your name** here like *I am Mohan Tummala* ...